### ISSUE TITLE
TYPO3 11 LTS support 

### ISSUE THREAD

**Issue by outdoorsman** [pfeilda/typo3samlauthentication#8]
We are wanting to upgrade to TYPO3 11.5 LTS and were wondering if you were planning on supporting version 11 and if so, when? Just don't want to start using this in v10 if there's no plans to update. Thanks for you extension :)

**Comment by pfeilda** [comment]
Hello, currently im a little bit busy. But I plan to raise the version until end of february or begin of March.

**Comment by pfeilda** [comment]
Is this helpfull for you?

**Comment by outdoorsman** [comment]
Yes thank you that is helpful. I will try testing this in v10 then and see if we can get it working... just having trouble the the `saml_miniorange` plugin and looking for an alternative. It's nice to have a community so we aren't just limited to one and only one option. Thank you for your work!

**Comment by jukkatawast** [comment]
Hi,
Any news regarding the compatibility with 11 LTS? 
Has anyone tried if the extension works if we just raise the version requirements.

I see that some bug fixes have been made during July, which is nice.

**Comment by pfeilda** [comment]
Sorry I was to bussy this year. But it is still planned. As we did some changes in the last week (thanks to @rvock) I will try to do it while my holiday.
